pkg/scanner/scanclientdownloader.go

```diff
@@ -84,6 +84,9 @@ func downloadScanClient(hubHost string, hubUser string, hubPassword string) (*sc
 	}
 	log.Infof("successfully unzipped from %s to %s", scanClientZipPath, scanClientRootPath)
 
+	javaScanClientJrePath := fmt.Sprintf("%s/scan.cli-%s/jre/", scanClientRootPath, currentVersion.Version)
```

Comment:
This is a super minor comment, but would you mind moving these lines to the `NewScanner` func?

The rationale is that there's already another environment variable being set in that function, and this code will be easier to understand if those settings are localized in the same place.